supertinycron/idf_component.yml

```diff
@@ -0,0 +1,5 @@
+version: "1.0.0~1"
```

Comment:
Should this be "2.0.0" to match the upstream version?

Reply:
Yes, correct it.